### PR TITLE
feat(azure): Add focal to jammy upgrade doc

### PR DESCRIPTION
**Pull request by AleksaSvitlica**
Add documentation for Focal to Jammy upgrade on Azure. Includes pictures showing the steps which require manual input.

### CODE REVIEW

File: docs/explanations/Azure/Focal_To_Jammy_Upgrade.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+Running [`do-release-upgrade`](https://manpages.ubuntu.com/manpages/focal/man8/do-release-upgrade.8.html) requires some manual intervention ([see Manual upgrade steps](#manual-upgrade-steps)), which makes it a good option for systems which require manual configuration and cannot be easily created or destroyed. 
+
+For system deployments which are fully automated it is recommended to redeploy with new Jammy instances instead of upgrading from Focal.
+
+## Manual upgrade steps
+If you are upgrading from Focal to Jammy you can expect to run into the following prompts requesting manual input.
+
+### Additional SSH daemon
+![Additional SSH daemon](/docs/explanations/Azure/Focal_To_Jammy_Upgrade_Images/0_additional_ssh_daemon.png)
```
Comment (j5awry) [inline docs/explanations/Azure/Focal_To_Jammy_Upgrade.md:14]: let's add short written descriptions under the heading, and alt-text for the images describing them

**Review by j5awry — CHANGES_REQUESTED** [review]
One general note is to keep accessibility in mind. let's make sure to add alt-text whenever we provide images. 

I'm also unsure about image paths -- should we have a separate assets folder for static assets that get shipped in?

File: docs/explanations/Azure/Focal_To_Jammy_Upgrade.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+![Firewall for additional SSH daemon](/docs/explanations/Azure/Focal_To_Jammy_Upgrade_Images/1_firewall_for_additional_ssh.png)
+
+### Start upgrade
+![Start upgrade](/docs/explanations/Azure/Focal_To_Jammy_Upgrade_Images/2_start_upgrade.png)
+
+### Restart services automatically
+![Restart services without asking](/docs/explanations/Azure/Focal_To_Jammy_Upgrade_Images/3_restart_services.png)
+
+### Chrony configuration modified
+![Chrony modified config](/docs/explanations/Azure/Focal_To_Jammy_Upgrade_Images/4_chrony_modified_config.png)
```
[the remark this answers is not in the document]
Comment (gjolly) [reply docs/explanations/Azure/Focal_To_Jammy_Upgrade.md:26]: Should we do it here or do we want to detail all the customization we do to azure images somewhere else?

**Comment by gjolly** [comment]
LGTM

**Comment by AleksaSvitlica** [comment]
Updated to include short descriptions for each heading and more descriptive alt-test for the images. What remains to be decided is:
- Are we okay with the image path or do we want them somewhere else?
- Do we want to provide any specific explanation about the config changes to SSHD and Chrony here or elsewhere?
- Do we have any specific advise for the UCF prompts for SSHD/Chrony? Should users keep the local configuration?

**Review by j5awry — APPROVED** [review]

**Comment by j5awry** [comment]
> Updated to include short descriptions for each heading and more descriptive alt-test for the images. What remains to be decided is:
> 
>     * Are we okay with the image path or do we want them somewhere else?
> 
The image paths are fine. I was talking to some others about it, and, really, each screenshot ends up unique. If we choose to move later, we can.

>     * Do we want to provide any specific explanation about the config changes to SSHD and Chrony here or elsewhere?
> 
I vote elsewhere. And to make those descriptions more verbose (the "why" it's that way, and our roadmap for changes)

>     * Do we have any specific advice for the UCF prompts for SSHD/Chrony? Should users keep the local configuration?
The default it is to keep the local config. I think pointing out the difference here is enough for now. Once we build pages for the Azure specific changes, it'll help people understand more fully. Short term, if they see this, it _should_ lead users to check the configs themselves and make the decision.

**Pull request merged**